config.yml

```diff
@@ -322,14 +333,14 @@ production_factories:
       - Bake_Bread
       - Bake_Cake
       - Bake_Pumpkin_Pie
-    repair_multiple: 26
+    repair_multiple: 10
     repair_inputs:
       Baked Potato:
         material: BAKED_POTATO
-        amount: 2
+        amount: 5
       Cookie:
         material: COOKIE
-        amount: 4
+        amount: 10
       Bread:
         material: BREAD
```

Comment:
I can't comment on whether this was intentional or not, but this is perfectly consistent with the latest changes in merge request #86 -- lines [334](https://github.com/Civcraft/FactoryMod/pull/86/files#diff-0f1be8eec416e71c28b41efb0544df26R334), 338, and 341. 

That said, I concur, I'll go ahead and make bread values consistent -- good catch! Seems we all missed it so far :D .